api/controller/helloworld_controller.go

```diff
@@ -0,0 +1,15 @@
+package controller
```

Comment:
No need to add the suffix `controller` to these files, the package/folder they're in gives enough context of what they are